### PR TITLE
BZ#1938953 - Add OpenShift SDN egress network policy limitation

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1938953

Somehow this got removed some time ago.

### CODE REVIEW

**Comment by jboxman** [comment]
@anuragthehatter, can you confirm? Thanks!

**Comment by jboxman** [comment]
@zhaozhanqi, can you take a look? Thanks!

File: modules/nw-networkpolicy-about.adoc [hunk 1]
```diff
@@ -11,7 +11,10 @@ In {product-title} {product-version}, OpenShift SDN supports using network polic
 
 [NOTE]
 ====
-IPBlock is supported by network policy with limitations for OpenShift SDN; it supports IPBlock without except clauses. If you create a policy with an IPBlock section that includes an except clause, the SDN pods log warnings and the entire IPBlock section of that policy is ignored.
+When using the OpenShift SDN cluster network provider, the following limitations apply regarding network policies:
+
+* Egress network policy as specified by the `egress` field is not supported.
+* IPBlock is supported by network policy with limitations for OpenShift SDN; it supports IPBlock without `except` clauses. If you create a policy with an IPBlock section that includes an `except` clause, the SDN pods log warnings and the entire IPBlock section of that policy is ignored.
```
Comment (mburke5678) [inline modules/nw-networkpolicy-about.adoc:17]: _IPBlock is supported by network policy with limitations for OpenShift SDN; it supports IPBlock without `except` clauses._
Can these two sentences be merged into one? It seems a bit clunky here. 
_Network policy for OpenShift SDN only supports IPBlock without `except` clauses._ (Or something like that)
#wordsarehard

**Comment by mburke5678** [comment]
@jboxman One suggestion for your consideration. Otherwise LGTM

**Comment by jboxman** [comment]
@mburke5678 thanks!

**Pull request merged**